package.json

```diff
@@ -87,11 +87,20 @@
       {
         "command": "mesonbuild.restartLanguageServer",
         "title": "Meson: Restart Language Server"
+      },
+      {
+        "command": "mesonbuild.selectRootDir",
+        "title": "Meson: Select another project root dir"
       }
     ],
     "configuration": {
       "title": "Meson build configuration",
       "properties": {
+        "mesonbuild.selectRootDir": {
+          "type": "boolean",
+          "default": true,
+          "description": "Should Meson ask to select a Meson project root directory when more than one project is detected."
```

Comment:
Remove the first two words of the description

Reply:
done